### PR TITLE
Docs updates

### PR DESCRIPTION
**Pull request by ozdanborne**
Corrected a lot of incorrect and old information

### CODE REVIEW

File: docs/mesos/ManualInstallCalicoMesos.md [hunk 1]
```diff
@@ -1,73 +1,48 @@
...
 >
-> View the calico-mesos-deployments documentation for the latest release [here](https://github.com/projectcalico/calico-mesos-deployments/blob/0.26.0%2B1/README.md).
+> View the calico-mesos-deployments documentation for the latest release [here](https://github.com/projectcalico/calico-mesos-deployments/blob/0.27.0%2B2/README.md).
 <!--- else
 > You are viewing the calico-mesos-deployments documentation for release **release**.
 <!--- end of master only -->
 
 # Manually Install Calico and Mesos on Agent
-This tutorial will walk you through installing Mesos, Netmodules, and Calico onto a Centos host to create a Mesos agent compatible with Calico.
+This guide explains how to add calico networking to a Mesos slave. Specifically, we will add the calico plugin to perform networking for the unified containerizer.
```
Comment (TrimBiggs) [inline docs/mesos/ManualInstallCalicoMesos.md:10]: calico -> Calico (2 here)

File: docs/mesos/ManualInstallCalicoMesos.md [hunk 2]
```diff
@@ -1,73 +1,48 @@
...
+- For information on the Docker Containerizer, see [Docker Containerizer Guide](#)
```
Comment (TrimBiggs) [inline docs/mesos/ManualInstallCalicoMesos.md:11]: This link doesn't work.

File: docs/mesos/ManualInstallCalicoMesos.md [hunk 3]
```diff
@@ -1,73 +1,48 @@
...
-
-```
-hostname -f
-```
-Then attempt to ping that name from other hosts.
 
-It is also important that Calico and Mesos have the same view of the (non-fully-qualified) hostname.  Ensure that the value returned by `hostname` is unique for each host in your cluster.
+## Prerequisites
+### Mesos-Slave with Mesos and Netmodules installed
+- This guide will focus on only the calico components of a calico-mesos-slave. You must have a slave with net-modules activated. See the [Add Netmodules to Mesos Guide](#)
```
Comment (TrimBiggs) [inline docs/mesos/ManualInstallCalicoMesos.md:16]: calico -> Calico

File: docs/mesos/ManualInstallCalicoMesos.md [hunk 4]
```diff
@@ -1,73 +1,48 @@
...
 
 ### Docker
-Since this is a dockerized deployment, you will need Docker installed on every Master and Agent in your cluster.
+Calico's core services are run in a docker container, so we'll need Docker installed on every Agent in the cluster.
 [Follow Docker's Centos installation guide](https://docs.docker.com/engine/installation/centos/) for information on how to get Docker installed.
 
+### Running etcd instance
+[prerequisites guide](prerequisites)
```
Comment (TrimBiggs) [inline docs/mesos/ManualInstallCalicoMesos.md:23]: Beef this.

File: docs/mesos/ManualInstallCalicoMesos.md [hunk 5]
```diff
@@ -1,73 +1,48 @@
...
-| mesos-agent  | 5051/tcp          |
-
-Example `firewalld` config:
-
-```
-sudo firewall-cmd --zone=public --add-port=179/tcp --permanent
-sudo firewall-cmd --zone=public --add-port=5051/tcp --permanent
-sudo systemctl restart firewalld
+## 1. Install Calico-Mesos Components
+The following scripts will add the necessary calico components.
```
Comment (TrimBiggs) [inline docs/mesos/ManualInstallCalicoMesos.md:26]: Calico

Comment (TrimBiggs) [inline docs/mesos/ManualInstallCalicoMesos.md:26]: Might want to briefly say what these components are also.

File: docs/mesos/MesosClusterPreparation.md [hunk 6]
```diff
@@ -1,68 +1,72 @@
...
 
-## 1. Launch ZooKeeper
-Mesos uses ZooKeeper to elect and keep track of the leading master in the cluster.
 
-```
-sudo docker pull jplock/zookeeper:3.4.5
-sudo docker run --detach --name zookeeper -p 2181:2181 jplock/zookeeper:3.4.5
-```
+## Prerequisite: Add the Mesos Official Repository
+Mesos-master, zookeeper, and marathon will all installed from the official Mesos repository. Add the official repository:
```
Comment (TrimBiggs) [inline docs/mesos/MesosClusterPreparation.md:21]: "will all installed"

File: docs/mesos/MesosClusterPreparation.md [hunk 7]
```diff
@@ -1,68 +1,72 @@
...
+## etcd
+Calico uses etcd as its data store and communication mechanism among Calico components. Install it from Centos' repositories with the following command, replacing `$IP` with the cluster-accessible IP you would like etcd to bind to:
+```
+yum install -y etcd
+echo ETCD_LISTEN_CLIENT_URLS=\"http://0.0.0.0:2379\" >> /etc/etcd/etcd.conf
+echo ETCD_ADVERTISE_CLIENT_URLS=\"http://$IP:2379\" >> /etc/etcd/etcd.conf
+systemctl start etcd.service
+```
 
+Etcd listens for tcp connections on ports 2379 and 4001. Open these ports on your firewall:
```
Comment (TrimBiggs) [inline docs/mesos/MesosClusterPreparation.md:69]: 2379 isn't mentioned below and is more important than 4001 considering the above change

File: docs/mesos/net-modules/BuildNetmodules.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
+# Install Net-modules
```
Comment (TrimBiggs) [inline docs/mesos/net-modules/BuildNetmodules.md:1]: Do we need this? Seems like a doc that should be in Mesos, not Calico.  This is like explaining how to build and install Docker from our repo.

Comment (ozdanborne) [inline docs/mesos/net-modules/BuildNetmodules.md:1]: Yeah I put a README in the net-modules folder that this doc is in, that disclaims "these docs will get moved to net-modules". I just thought temporarily we could house them while we revise them, then PR them into net-modules itself

Comment (TrimBiggs) [inline docs/mesos/net-modules/BuildNetmodules.md:1]: Gotcha

**Pull request closed without merging**